packages/ag-charts-community/src/util/object.ts

```diff
@@ -131,3 +131,22 @@ export function deepFreeze<T>(obj: T): T {
 
     return obj;
 }
+
+export function defineSetterGetter<T, P extends string, V>(
+    o: T,
+    p: P,
+    attributes: { set: (v: V) => void; get: () => V }
+): T & { [K in P]: V };
+export function defineSetterGetter<T, P extends string, V>(
+    o: T,
+    p: P,
+    attributes: { set?: undefined; get: () => V }
```

Comment:
This looks wrong - probably we should either:
- Omit `set` here.
- Make it `unknown` or `never` rather than `undefined`.

Reply:
It's not wrong. The overload simply states that the type of the `set` is undefined, and that it can be explicitly set to undefined or implicitly set to by omitting the property.

However, it seems that `never` accomplishes the same thing so I'll try with that.

Reply:
obsolete.